test_bin/data/test_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "N3H_PATH": "C:\\github\\n3h",
+  "modes": {
+    "IN_MEMORY": false,
+    "IPC_MOCK": false,
+    "HACK_MODE": true
```

Comment:
if this is the default one, do we need `IN_MEMORY` to be the only one that is true for CI?

Reply:
The default file is not used by CI.
CI is calling IN_MEMORY for all suites directly.
This is more like the default file for testing locally and debugging.
I did update some values since your comment.